alacritty/src/renderer/mod.rs

```diff
@@ -1088,24 +1102,25 @@ impl<'a, C> RenderApi<'a, C> {
 ///
 /// If the current atlas is full, a new one will be created.
 #[inline]
-fn load_glyph(
-    active_tex: &mut GLuint,
-    atlas: &mut Vec<Atlas>,
-    current_atlas: &mut usize,
-    rasterized: &RasterizedGlyph,
-) -> Glyph {
-    // At least one atlas is guaranteed to be in the `self.atlas` list; thus
-    // the unwrap.
+fn load_glyph(layers: &mut Vec<Layer>, rasterized: &RasterizedGlyph) -> Glyph {
+    let (layer, atlas_type) = &mut match rasterized.buf {
+        BitmapBuffer::RGB(_) => (&mut layers[0], gl::RGB),
+        BitmapBuffer::RGBA(_) => (&mut layers[1], gl::RGBA),
```

Comment:
Hardcoding which atlas type uses which color format here seems like it could cause troubles in future refactorings.

Should the colorspace maybe be stored in the layer? Or/And the layer should maybe be indexed by layer type or colorspace?